### ISSUE TITLE
Getting Started

### ISSUE THREAD

**Issue by c-sonnier** [wescale/hashistack#3]
I've run through the getting started and setup my inventory.ini but what command do I need to run to start the deploy phases?

**Comment by aurelienmaury** [comment]
Hi, I realize how much time has passed since the opening of this issue. I've been away from this codebase foir personnal reasons and I only saw it 3 months ago. Be sure I am sorry for that. We are actively working on improving the capabilities and the onboarding.

If you are inclined to give a second try, please have a careful look at this updated tutorial : https://hashistack.readthedocs.io/en/latest/tutorials/deploy_scw.html

Also, if you are not deploying on Scaleway, please let me know it right before trying and I will prioritize updating the documentation to give you the technical hints to a successful deployment.

Regards

**Comment by c-sonnier** [comment]
No worries, I completely understand. I was not planning on deploying to Scaleway. I run my own ESXi server and was going to run Hashistack on a few Debian VMs. Looking forward to trying to get this to run.

**Comment by aurelienmaury** [comment]
By next week, I'll provide some hints in the documentation to ease your path in replacing the stage0 (infrastructure) by your own and validate that everything is ok before starting stage1.
Without being too curious, will you rely on our LetsEncrypt automation (implying a public DNS delegation and exposure) for certificates on supply some prepared by your own mean?

**Comment by c-sonnier** [comment]
Haven't decided. I do run nginx proxy manager in front of all my websites which handles my LetsEncrypt certs, but not sure  how Hashistack behaves behind a nginx reverse proxy.

**Comment by aurelienmaury** [comment]
Well actually we handle a nginx reverse proxy as front server to load balance to vault, consul and nomad admin webUI and to nomad minions nodes. Adaptation should be quite feasable if you base yourself on our configuration template for examples.

I added the promised documentation [right there](https://hashistack.readthedocs.io/en/latest/howto/supply_infra.html). I will close this issue but please feel free to open some more when you decide to get your hands dirty.

Again: Thanks for your interest.

**Comment by c-sonnier** [comment]
Thank you very much for taking the time to add the documentation. If I come across any road blocks, I will be sure to open another issue. Looking forward to getting things up and running.
